spearhead/assets/sass/_forms.scss

```diff
@@ -0,0 +1,19 @@
+input[type="text"], 
+input[type="email"], 
+input[type="url"], 
+input[type="password"], 
+input[type="search"], 
+input[type="number"], 
+input[type="tel"], 
+input[type="range"], 
+input[type="date"], 
+input[type="month"], 
+input[type="week"], 
+input[type="time"], 
+input[type="datetime"], 
+input[type="datetime-local"], 
+input[type="color"], 
+textarea {
+    background: transparent;
+    box-shadow: none;
+}
```

Comment:
I wonder if it would be better to add this to Seedlet as variables that we can override?